katello/katello-debug.sh

```diff
@@ -81,14 +87,10 @@ add_cmd "qpid-stat --ssl-certificate=/etc/pki/katello/qpid_client_striped.crt -b
 add_files /etc/gofer
 add_files /var/log/gofer
 
-#foreman-tasks export
-if hash foreman-rake 2>/dev/null; then
-  echo "Exporting tasks, this may take a few minutes."
-  tasks_filename=`foreman-rake foreman_tasks:export_tasks 2> /tmp/tasks_export.log | tail -n 1 | awk '{print $2}'`
-  copy_files $tasks_filename
-  add_files /tmp/tasks_export.log
-  rm -f $tasks_filename /tmp/tasks_export.log
-fi
+# Foreman Tasks fast export (for HTML version use foreman-rake foreman_tasks:export_tasks)
+add_cmd "echo \"select dynflow_execution_plans.* from foreman_tasks_tasks join dynflow_execution_plans on (foreman_tasks_tasks.external_id = dynflow_execution_plans.uuid) where foreman_tasks_tasks.started_at > 'now'::timestamp - '2 months'::interval and foreman_tasks_tasks.result = 'success'\" | su foreman -s /usr/bin/psql -- -t -A -F',' foreman" dynflow_execution_plans.csv
```

Comment:
I also suggest exporting `"select * from dynflow_schema_info"`, to have better info about version of schema it was exported from